mastering-plone/testing.rst

```diff
@@ -139,13 +139,13 @@ Getting started
 Mr.bob already created the testing layers.
 We will go through them now.
 
-Next, it adds a method for testing that your add-on gets properly installed. This might seem stupid, but it isn't if you take into account that in plone land, things change with new releases. Having a GenericSetup profile installing Javascript files contains the assumption that the package wants a javascript file available in Plone.
-This assumption is explained in the syntax of the current Plone. By testing that the result is met, the Javascript file really is available, we spell out that assumption more clearly.
-The person that wants to make your package work 5 years from now, knows now that the result in his browser might be related to a missing file. Even if he does not understand the semantics from the old Plone on how to register js files, he has a good starting point on what to do to make this package compatible.
+Next, it adds a method for testing that your add-on gets properly installed. This might seem stupid, but it isn't if you take into account that in Plone land, things change with new releases. Having a GenericSetup profile installing JavaScript files contains the assumption that the package wants a JavaScript file available in Plone.
+This assumption is explained in the syntax of the current Plone. By testing that the result is met (that the JavaScript file really is available), we spell out that assumption explicitly.
+The person that wants to make your package work 5 years from now, knows now that the result in her browser might be related to a missing file. Even if she does not understand the semantics from the old Plone on how to register JavaScript files, she has a good starting point on what to do to make this package compatible.
```

Comment:
The gender neutral pronoun (they, their) is preferred over gender-specific.

The use of "he/she" shall be punished severely by being slapped with Kraft American singles cheese food.

Reply:
I admit, it makes an excellent roofing material.